src/Illuminate/Database/Eloquent/Model.php

```diff
@@ -1128,7 +1128,7 @@ public function newCollection(array $models = [])
     /**
      * Create a new pivot model instance.
      *
-     * @param  \Illuminate\Database\Eloquent\Model  $parent
+     * @param  self  $parent
```

Comment:
Please don't change this. It is a stylistic choice to use the full name in the phpdoc.